docs/source/guides/ethereum_node.rst

```diff
@@ -163,7 +163,7 @@ Clef is typically installed alongside geth.
 
 .. important::
 
-    Geth version 1.9.0 or higher is required.
+    Geth version 1.9.22 or higher is required.
```

Comment:
What are your thoughts regarding adding something like this to the release notes for the corresponding version that introduced this requirement i.e. editing `docs/source/release_notes/releases.rst` directly?

It could be added as an `.. important::` element, or as a bullet point in an added `Deprecations and Removals` section.

Reply:
I think it's probably not worth it ... According to https://www.ethernodes.org/client/geth, geth >= 1.9.22 accounts for 90% of geth nodes.